### ISSUE TITLE
No tokens were issued after the victory

### ISSUE THREAD

**Issue by NexGen3** [ValveSoftware/Dota2-Gameplay#18222]
### Description

Tonight, after the game, due to broken servers, the results of the game did not appear immediately. When I entered the game in the morning, I found that the game was counted and no tokens were credited. Is it possible for me to enroll them somehow??

### Screenshots

![Снимок экрана 2024-05-24 141511](https://github.com/ValveSoftware/Dota2-Gameplay/assets/170725410/0725120c-8ed7-4e22-9f33-ade1f60018ec)
![Снимок экрана 2024-05-24 141614](https://github.com/ValveSoftware/Dota2-Gameplay/assets/170725410/3d690747-13a4-4a4e-91e9-f23c38bd9c22)

**Comment by lannahirave** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/77197415/50e280cb-7024-4ed9-b23c-bb2810a863d9)
I bet that's because the event is not selected as ACTIVE in main menu. Just got the tokens for the first act...

**Comment by NexGen3** [comment]
Indeed, they gave tokens for Act 1, although Act 2 was selected at the time of passing. I didn't take it out of my head that I needed to play a certain character.

**Comment by NexGen3** [comment]
Will they help me do everything properly, or will they turn a blind eye to their problems with the servers and say that it's my fault?
